unix/vncserver/vncserver.in

```diff
@@ -223,6 +236,7 @@ if ($cookie eq "") {
   }
 }
 
+$xauthorityFile = $config{'auth'} || $xauthorityFile;
```

Comment:
This seems to be an independent change? Let's try to focus on just one thing in this PR.